doc/application/index.rst

```diff
@@ -1522,6 +1522,173 @@ After running the preprocessor, the final device tree used in the
 build is created by running the device tree compiler, ``dtc``, on the
 preprocessor output.
 
+Building and Configuring multiple images
+****************************************
+
+When firmware update and bootloaders are involved in an embedded
+application the final firmware that is flashed onto the device will
+usually need to consist of not one executable, but multiple
+executables that chain-load, or boot, each other. This section
+describes how to build and configure such a Zephyr application.
+
+When and why to use multiple images
+===================================
+
+First off, it's necessary to have a basic understanding of what
```

Comment:
Let's simplify the introduction to:
```
An executable goes by other names depending on context, for example:
program, image, or elf file. (From now on we will use the name
image.)  An image consists of pieces of code and data that
go by image-unique names recorded in a single symbol table.
The symbol table ...
```

Reply:
I've re-written it to be:

> An executable goes by other names depending on context, for example:
> image, or elf file. (From now on we will use the name image, short for
> "executable program image".)  An image consists of pieces of code and
> data that go by image-unique names recorded in a single symbol table.

This:

Cuts the sentences that David didn't like.

Removes the word "program".

Clarifies that in the following text, the word 'image' is short for "executable program image". Which I hope addresses your concerns Marc.

I can't find the next comment.